packages/textlint-config/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@strv/textlint-rule-preset-strv",
+  "description": "STRV's rule preset for textlint",
+  "version": "1.0.0",
+  "author": "Korneliusz Dubieniecki <lekterable@gmail.com>",
+  "bugs": {
+    "url": "https://github.com/strvcom/code-quality-tools/issues"
+  },
+  "contributors": [],
+  "dependencies": {
+    "textlint": "^11.2.5",
```

Comment:
Should `textlint` itself really be a direct dependency of a rule preset? 🤔 Perhaps it should be a `peerDependency` only? I would expect that `textlint` be installed on a per-project basis and not as part of this rule preset.